addons/website/static/src/snippets/s_timeline/000.scss

```diff
@@ -1,4 +1,4 @@
-.s_timeline {
+.s_timeline:not([data-vcss]) {
     --spacing: #{$grid-gutter-width * .5};
```

Comment:
@Brieuc-brd Sorry I just remembered I had a last question that I forgot to ask:
We agree that it is okay if we are not able to place two contents with cards next to each other anymore (like in the old snippet) ? Because there is no way to add new events excepts by duplicating the rows, so previously, if a user wanted two events at the same date, he could duplicate the line with two events on it, but now it is completely impossible :open_mouth: 
Was this wanted/discussed before ? :slightly_smiling_face: 
At worst, if people complain, we could add an option later, but allowing to duplicate from the templates was a simple and easy workaround to this issue :wink: